src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -276,7 +276,7 @@ private void InstallButton_Clicked(object sender, RoutedEventArgs e)
 
         private void Versions_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            PackageDetailControlModel.PreviousSelectedVersion = e.AddedItems.Count > 0 && !e.AddedItems[0].Equals(null) ? e.AddedItems[0].ToString() : string.Empty;
+            PackageDetailControlModel.PreviousSelectedVersion = e.AddedItems.Count > 0 && e.AddedItems[0] != null ? e.AddedItems[0].ToString() : string.Empty;
```

Comment:
> Code is in the UI code behind. Manually tested by attaching a debugger, turning on exceptions, setting a breakpoint, and validating that when AddedItems[0] is null, an NRE is not thrown.

Can the method's access modifier be changed to `internal`, and then a unit test written to minimize risk of future regressions?

Reply:
> > Code is in the UI code behind. Manually tested by attaching a debugger, turning on exceptions, setting a breakpoint, and validating that when AddedItems[0] is null, an NRE is not thrown.
> 
> Can the method's access modifier be changed to `internal`, and then a unit test written to minimize risk of future regressions?

We chatted offline and are satisfied with the test exception. This scenario was not regressed but rather the first attempted fix continued to cause an NRE which I noticed while debugging. A unit test could have caught the issue but there are several challenges, mainly that WPF components are not unit testable so this logic should live in a separate layer which is out of scope for this change.